app/js/index.js

```diff
@@ -2,7 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import Navbar from './components/navbar';
+import Event from './components/event';
+import Ticker from './components/ticker';
+
+//const {Route, RouteHandler} = Router;
```

Comment:
Rebase against master - this was deleted, but is now an addition of `//const {Route, RouteHandler} = Router;`